dist/templates/newsletters/callout-example.html

```diff
@@ -0,0 +1,297 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<title>ACLU Email</title>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+		<meta name="viewport" content="width=device-width" />
+		<meta name="color-scheme" content="light dark" />
+		<meta name="supported-color-schemes" content="light dark" />
+		<style>
+			:root {
+				color-schemes: light dark;
+				supported-color-schemes: light dark;
+			}
+			h1,
+			h2,
+			h3,
+			h4,
+			h5,
+			h6 {
+				margin: 0 0 5px 0;
+			}
+
+			p {
+				margin-top: 0px;
+				margin-bottom: 20px;
+			}
+
+			body {
+				width: 100% !important;
+				-webkit-text-size-adjust: 100%;
+				-ms-text-size-adjust: 100%;
+				margin: 0px;
+				padding: 0px;
+			}
+
+			a,
+			a:visited,
+			a:hover,
+			a:active {
+				color: #0055aa;
+				font-weight: 900;
+			}
+
+			img {
+				outline: none;
+				text-decoration: none;
+				-ms-interpolation-mode: bicubic;
+			}
+
+			a img {
+				border: none;
+			}
+
+			/* REMOVE SPACING BETWEEN TABLES IN OUTLOOK 2007 AND UP */
+			table,
+			td {
+				border-spacing: 0px;
+				border-collapse: collapse;
+				mso-table-lspace: 0pt;
+				mso-table-rspace: 0pt;
+			}
+
+			.preheader {
+				display: none !important;
+				visibility: hidden;
+				opacity: 0;
+				color: transparent;
+				height: 0;
+				width: 0;
+			}
+
+			ul {
+				margin-top: 0px;
+				margin-bottom: 0px;
+			}
+
+			li {
+				margin-bottom: 15px;
+				font-family: Arial, "Helvetica Neue", sans-serif;
+				font-size: 16px;
+				line-height: 22px;
+				/* REMOVE list-style-type
+	TO SHOW ORDERED LIST */
+				list-style-type: disc;
+			}
+
+			td.footer-text a {
+				font-weight: 400 !important;
+			}
+
+			.highlight {
+				background-color: #f3ef07;
+				display: inline;
+				padding: 3px;
+				font-weight: bold;
+				color: black;
+			}
+
+			ul li::before,
+			ol li::before {
+				content: "\2022";
+				color: #f3ef07;
+				font-weight: bold;
+				display: inline-block;
+				width: 1em;
+				margin-left: -1em;
+			}
+
+			tr.header-section {
+				display: block;
+				padding: 20px 0;
+			}
+
+			td.logo {
+				width: 450px;
+			}
+			td.donate {
```

Comment:
Did you add this in? If so then place a style tag at the top of your component. Then I can pull out the code and add it to an single styles file that will pull in

Reply:
You can ignore if this was not you